toolflow/vivado/platform/sim/sim.tcl

```diff
@@ -0,0 +1,426 @@
+# Copyright (c) 2014-2020 Embedded Systems and Applications, TU Darmstadt.
+#
+# This file is part of TaPaSCo
+# (see https://github.com/esa-tu-darmstadt/tapasco).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+namespace eval platform {
+  set platform_dirname "sim"
+
+  if { [::tapasco::vivado_is_newer "2018.1"] == 0 } {
```

Comment:
What was the reason for this?

Reply:
Since most of the simulation platform is based on the zynqmp platform, i have copied this statement aswell. The Zynqmp is not compatible with versions older than 2018.1 and the simulations underlying Xilinx part is the same as the Ultra96v2 and in turn the ZynqMP.

Reply:
ok